controllers/chat/group/groupAddMembers.js

```diff
@@ -68,21 +69,39 @@ const groupAddMembers = async (req, res) => {
       const textTargetUser = `${ownUser.username} added you to this group`;
       const textDefaultUser = `${ownUser.username} added ${targetUserModel.username} to this group`;
 
-      await channelToAdd.sendMessage({
-        text: textDefaultUser,
-        own_text: textOwnUser,
-        other_text: textTargetUser,
-        other_system_user: member,
-        better_type: 'add_member_to_group',
-        type: 'system',
-        user_id: userId,
-        only_to_user_show: userId,
-        disable_to_user: false,
-        is_from_prepopulated: true,
-        system_user: userId,
-        isSystem: true,
-        members: [member]
-      });
+      await channelToAdd.sendMessage(
+        {
+          text: textDefaultUser,
+          own_text: textOwnUser,
+          other_text: textTargetUser,
+          other_system_user: member,
+          better_type: 'add_member_to_group',
+          type: 'system',
+          user_id: userId,
+          only_to_user_show: userId,
+          disable_to_user: false,
+          is_from_prepopulated: true,
+          system_user: userId,
+          isSystem: true,
+          members: [member]
+        },
+        {
+          skip_push: true
+        }
+      );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding `skip_push: true` to the message sending call will prevent push notifications from being sent when a message is added. This is a significant change in behavior and should be thoroughly tested to ensure it doesn't negatively impact user experience.



<!-- This is an auto-generated comment by OSS CodeRabbit -->